frameworks/keyed/ember/app/utils/benchmark-helpers.js

```diff
@@ -31,25 +28,25 @@ export const buildData = (id, count = 1000) => {
       "table", "chair", "house", "bbq", "desk", "car", "pony", "cookie",
       "sandwich", "burger", "pizza", "mouse", "keyboard"];
 
-    var data = [];
+    var data = A();
```

Comment:
should we be using tracked properties and native arrays and objects?

Reply:
@NullVoxPopuli  can't get it working in stable release